invisible_cities/cities/isidora_test.py

```diff
@@ -0,0 +1,124 @@
+from   invisible_cities.database import load_db
+from   invisible_cities.cities.isidora import Isidora
+from   invisible_cities.core.random_sampling \
+     import NoiseSampler as SiPMsNoiseSampler
+
+config_file_format = """
+# set_input_files
+PATH_IN {PATH_IN}
+FILE_IN {FILE_IN}
+
+# set_cwf_store
+PATH_OUT {PATH_OUT}
+FILE_OUT {FILE_OUT}
+COMPRESSION {COMPRESSION}
+
+# irene
+RUN_NUMBER {RUN_NUMBER}
+
+# set_print
+NPRINT {NPRINT}
+
+# set_blr
+NBASELINE {NBASELINE}
+THR_TRIGGER {THR_TRIGGER}
+
+# set_mau
+NMAU {NMAU}
+THR_MAU {THR_MAU}
+
+# run
+NEVENTS {NEVENTS}
+RUN_ALL {RUN_ALL}
+"""
+
+config_file_spec = dict(PATH_IN  = '$ICDIR/database/test_data/',
+                        FILE_IN  = 'electrons_40keV_z250_RWF.h5',
+                        PATH_OUT = '$ICDIR/database/test_data/',
+                        FILE_OUT = 'cwf_electrons_40keV_z250_RWF.h5',
+                        COMPRESSION = 'ZLIB4',
+                        RUN_NUMBER  = 0,
+                        NPRINT      = 1,
+                        NBASELINE   = 28000,
+                        THR_TRIGGER = 5,
+                        NMAU        = 100,
+                        THR_MAU     = 3,
+                        NEVENTS     = 5,
+                        RUN_ALL     = False)
+
+config_file_contents = config_file_format.format(**config_file_spec)
+
+@mark.slow
+def test_isidora_run(config_tmpdir):
+    """Test that Isidora runs on default config parameters.
+       Check that CWF waveforms math BLR within 1%. """
+
+
+    conf_file_name = str(config_tmpdir.join('test.conf'))
+    with open(conf_file_name, 'w') as conf_file:
+        conf_file.write(config_file_contents)
+
+    RWF_file = path.join(os.environ['ICDIR'],
+                          'database/test_data/electrons_40keV_z250_RWF.h5')
+
+    CWF_file = str(config_tmpdir.join(
+                   'electrons_40keV_z250_CWF.h5'))
+    CFP = configure(['ISIDORA',
+                     '-c', conf_file_name])
+
+    RWF_file = CFP['FILE_IN']
+    CWF_file = CFP['FILE_OUT']
+
+    fpp = Isidora(run_number=CFP['RUN_NUMBER'])
+    files_in = glob(RWF_file)
+    files_in.sort()
+    fpp.set_input_files(files_in)
+    fpp.set_cwf_store(CWF_file, compression = CFP['COMPRESSION'])
+    fpp.set_print(nprint = CFP['NPRINT'])
+    fpp.set_BLR(n_baseline  = CFP['NBASELINE'],
+                thr_trigger = CFP['THR_TRIGGER'] * units.adc)
+    fpp.set_MAU(  n_MAU = CFP['NMAU'],
+                thr_MAU = CFP['THR_MAU'] * units.adc)
+
+    nevts = CFP['NEVENTS'] if not CFP['RUN_ALL'] else -1
+    nevt = fpp.run(nmax=nevts)
+    assert nevt == nevts
+
+    eps = 1
+    with tb.open_file(RWF_file, 'r') as h5in, tb.open_file(CWF_file, 'r') as h5out:
+        pmtblr = h5in.root.RD.pmtblr
+        pmtcwf = h5out.root.BLR.pmtcwf
+        for event in range(nevt):
+            BLR = pmtblr[event]
+            CWF = pmtcwf[event]
+            for pmt in range(len(BLR)):
+                diff = abs(np.sum(BLR[pmt][4950:5150]) \
```

Comment:
Perhaps it would be helpful to replace the 4950 and 5150 *magic numbers* with meaningfully named variables?